ec2_security_groups_dumper/main.py

```diff
@@ -275,31 +286,45 @@ def _get_rules_from_aws(self):
                 group_dict['rules'] = list()
 
             for rule in group.rules:
-                rule_dict = dict()
-                rule_dict['ip_protocol'] = rule.ip_protocol
-                rule_dict['from_port'] = rule.from_port
-                rule_dict['to_port'] = rule.to_port
+		rule_dict = self._build_rule( rule )
+		rule_dict['direction']="INGRESS"
 
-                if rule.grants:
-                    rule_dict['grants'] = list()
-
-                for grant in rule.grants:
-                    grant_dict = dict()
-                    if grant.name:
-                        grant_dict['name'] = grant.name
-                    if grant.group_id:
-                        grant_dict['group_id'] = grant.group_id
-                    if grant.cidr_ip:
-                        grant_dict['cidr_ip'] = grant.cidr_ip
+                group_dict['rules'].append(rule_dict)
 
-                    rule_dict['grants'].append(grant_dict)
+            if group.rules_egress:
+                for rule in group.rules_egress:
+                    rule_dict = self._build_rule( rule )
+                    rule_dict['direction']="EGRESS"
+                        
+                    group_dict['rules'].append(rule_dict)
```

Comment:
`group_dict['rules']` is undefined if you don't have any VPC rule.
Can you test your PR in both scenario?